zebra-chain/src/block/tests.rs

```diff
@@ -1,11 +1,15 @@
+use chrono::{DateTime, NaiveDateTime, TimeZone, Utc};
 use std::io::{Cursor, ErrorKind, Write};
 
-use chrono::NaiveDateTime;
 use proptest::{
     arbitrary::{any, Arbitrary},
     prelude::*,
 };
 
+use crate::equihash_solution::EquihashSolution;
+use crate::merkle_tree::MerkleTreeRootHash;
+use crate::note_commitment_tree::SaplingNoteTreeRootHash;
+use crate::serialization::{ReadZcashExt, SerializationError, ZcashDeserialize, ZcashSerialize};
```

Comment:
Once this warning is cleaned up I think this PR is good to merge

```suggestion
use crate::serialization::{SerializationError, ZcashDeserialize, ZcashSerialize};
```